diagnostic.md

```diff
@@ -185,4 +234,8 @@ with the class `.stretch` be at a _screen width_ of 550px?
 
 <!-- your answer starts here -->
 
+550px as a screen width is bigger than the requested min-width of 400px from the first @media request but smaller necessary for the second once, so the first would apply.
+
+--> the width would be 25% from the original width that we can see at .strech (100px) --> The element would be 75px big.
```

Comment:
`@media( min-width= 400px)` means above this. From this, the 25% would apply. So at 550px, 25% would be `137.5px`